Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -15,70 +16,105 @@ class BRBViewController: UIViewController {
         case account(BRBAccount)
     }
     
-    private var connectionHandler: BRBConnectionHandler!
+    var accountManager: BRBAccountManager!
```

Comment:
Initialize inline with let to be more typesafe

Reply:
^